src/assets/css/solen.css

```diff
@@ -191,6 +191,12 @@ table thead th:first-child {
   justify-content: center;
 }
 
+#draft-bar {
+  margin: 1rem 0;
+  padding: 1rem;
+  background-color: #fea;
+}
+
```

Comment:
Ce serait chouette d'avoir des règles CSS pour un `cursor: not-allowed;` sur les select et radio aussi

Reply:
Je suis moins fan de ce curseur, mais on peut tenter si tu penses que ça ajoute de la valeur